ci/images/ci-runner/Dockerfile

```diff
@@ -27,6 +27,13 @@ RUN set -x \
         tar-2:1.34 \
         xz-5.2.5 \
     && microdnf clean all
+#Installing helm as part of a work around for installing the minio operator while we sort out stale operator hub catalogs with hypershift.
```

Comment:
This is not how we install software.
1. You need to pin a specific version in `shared/config/dependencies.sh`.
2. You need to add the command(s) to install the binary in `shared/hack/install`.
3. You need to add the binary to the `install.sh` command in the Dockerfile(s).